client/main.css

```diff
@@ -972,9 +973,12 @@ audio {
   margin: 0;
   background-color: #262626;
   color: white;
+  width: calc(100% - 60px);
 }
-.show__link a:hover, .event__link a:hover, .review__link a:hover, .chart__link a:hover,
-.show__link a, .event__link a, .review__link a, .chart__link a {
+.show__link a:hover, .event__link a:hover, .review__link a:hover,
+.chart__link a:hover, .playlist__link a:hover,
+.show__link a, .event__link a, .review__link a, .chart__link a,
+.playlist__link a {
```

Comment:
We should just make these __link classes all the same class. Would you mind patterning this and the __header classes according to that pattern?